contents/docs/new-to-posthog/revenue.mdx

```diff
@@ -0,0 +1,102 @@
+
+If we've got an upsell or call to action that lists a specific price, that's a great property to include in the event properties
+
+### Option 2 – Automating revenue capture with data warehouse
+
+Even at the earliest stages of a business, data can be scattered between multiple services, forcing you to constantly open multiple browser tabs just to keep up, which means your events might not give you all the data you need. 
+
+Instead of setting up complex event-sending pipelines, you can bring the data directly into PostHog's data warehouse and analyze it all from there.
+
+PostHog's **data warehouse** solves this problem by aggregating multiple data sources. Your dashboards can then report everything captured from data warehouse and analytics events in a single view. You can even build insights that traverse both.
+
+You can query *anything* from your app database with [data warehouse](https://posthog.com/docs/data-warehouse/query). Data warehouse also provides quick setup for ongoing capture of revenue events from [Stripe](https://posthog.com/docs/cdp/sources/stripe) and [Chargebee](https://posthog.com/docs/cdp/sources/chargebee). 
+
+Just don't forget to capture the actions *leading* to revenue.
+
+<Admonition icon="IconPencil" title="Planning revenue reporting">
+
+How will we report revenue?
+
+- Data warehouse?
+- Manual events?
+
+Which events are revenue-generating? Are they already tracked, or do we need to add additional tracking to capture them?
+
+</Admonition>
+
+## Revenue reporting
+
+With revenue logged, we can build reports to understand how much revenue is captured over time. This is a great way to understand the health of a business.
+
+A **trends** insight is a great start for your revenue reporting:
+
+- Create a new trends insight  
+- Edit the **series** to choose your revenue-bearing event  
+- In the aggregation menu to the right of the selected event, choose **property value > sum**  
+- In the rightmost menu, choose the property that captures the revenue from your event
+
+It will look like something like this:
+
+![revenue series configuration](https://res.cloudinary.com/dmukukwp6/image/upload/revenue_series_configuration_cfe291c330.png)
+
+
+From here we can choose the timespan for the report, grouping anywhere from hourly to monthly, with comparisons to the previous period if helpful.
+
+## Your product in one dashboard
+
+![a complete dashboard](https://res.cloudinary.com/dmukukwp6/image/upload/hogflix_c681b6891d.png)
+
+With revenue and all your key events captured, you can build a dashboard like this that covers:
+
+- Growth of daily, weekly, or monthly active users
+- Conversion from sign up to activation
+- Retention of users after they activate
+- Revenue trends relative to your goals
+
+In other words, everything that's important to your business.
+
+## Next steps
```

Comment:
Some CTA to understanding PostHog here?

Reply:
I'm kind of thinking Understanding Posthog needs to be relocated anyway. Could go into the integration docs above, I suspect, but it's very V0.5 atm. I think we leave it as is for now and promote it more once we've developed it further.